src/index.html

```diff
@@ -7,28 +7,61 @@ <h1>Instant, themed Twitch chat overlays</h1>
 	<form id="overlay-builder">
 		<label for="channel">Channel name</label>
 		<input type="text" id="channel" name="channel" required="true" autocomplete="off" autocorrect="off"
-			autocapitalize="off" spellcheck="false" />
-		<label for="theme">Theme</label>
+			autocapitalize="off" spellcheck="false" placeholder="{{defaults.channel}}"
+			title="Your Twitch channel name" />
+
+		<label for="theme">Theme <a href="docs/configuration/#theme">?</a></label>
 		<select id="theme" name="theme">
-			<option value="default">Default</option>
+			<option value="{{defaults.theme}}">{{defaults.theme | formatThemeName}}</option>
 			<option value="light">Light</option>
 			{%- for theme in themes.list -%}
-			{%- unless theme == "default" or theme =="light" -%}
+			{%- unless theme == "default" or theme == "light" -%}
 			<option value="{{ theme }}">{{ theme | formatThemeName }}</option>
-			{%- endunless -%} {%- endfor %}
+			{%- endunless -%}
+			{%- endfor %}
+		</select>
+
+		<label for="demo">Demo page <a href="docs/configuration/#demo">?</a></label>
```

Comment:
Thinking about these `?` links for the docs. I think the foundation is good, but there's probably a few things that we should do with them to ensure they're accessible:

First, we should move the `?` links outside the `<label>` elements. This is because it's typically considered inaccessible to nest clickable elements, and because I'd rather question marks not be used as part of the form fields' accessible names for assistive technologies.

Second, I think we should apply an `aria-label` to the anchor tags to make it clear what the purpose of these links is. I'd suggest `aria-label="Docs for DEMO"` or `aria-label="Docs for showLatestMessages"` and so forth.